apps/guide/src/content/04-slash-commands/01-command-response-methods.mdx

```diff
@@ -0,0 +1,354 @@
+				username: 'User',
+			},
+			command: '/interaction',
+		}}
+	>
+		Pong!
+	</DiscordMessage>
+	<DiscordMessage
+		author={{
+			avatar: '/assets/discordjs.png',
+			bot: true,
+			time: 'Today at 21:01',
+			username: 'Guide Bot',
+		}}
+		reply={{
+			author: {
+				avatar: '/assets/discordjs.png',
+				bot: true,
+				username: 'Guide Bot',
+			},
+			content: 'Pong!',
+		}}
+		time="21:02"
+	>
+		Pong again!
+	</DiscordMessage>
+</DiscordMessages>
+
+You can also pass an `ephemeral` flag to the `InteractionReplyOptions`:
+
+<CH.Code lineNumbers={false}>
+
+```js
+await interaction.followUp({ content: 'Pong again!', ephemeral: true });
+```
+
+</CH.Code>
+
+<DiscordMessages>
+	<DiscordMessage
+		author={{
+			avatar: '/assets/discordjs.png',
+			bot: true,
+			time: 'Today at 21:03',
+			username: 'Guide Bot',
+		}}
+		interaction={{
+			author: {
+				avatar: '/assets/discordjs.png',
+				username: 'User',
+			},
+			command: '/interaction',
+		}}
+	>
+		Pong!
+	</DiscordMessage>
+	<DiscordMessage
+		author={{
+			avatar: '/assets/discordjs.png',
+			bot: true,
+			time: 'Today at 21:01',
+			username: 'Guide Bot',
+		}}
+		reply={{
+			author: {
+				avatar: '/assets/discordjs.png',
+				bot: true,
+				username: 'Guide Bot',
+			},
+			content: 'Pong!',
+		}}
+		time="21:02"
+	>
+		Pong again!
+	</DiscordMessage>
+</DiscordMessages>
+
+Note that if you use <DocsLink type="class" parent="ChatInputCommandInteraction" symbol="followUp" brackets /> after a <DocsLink type="class" parent="ChatInputCommandInteraction" symbol="deferReply" brackets />, the first follow-up will edit the "_`<application> is thinking`_" message rather than sending a new one.
+
+That's all, now you know everything there is to know on how to reply to slash commands!
+
+<Alert title="Masked links" type="success">
+	Interaction responses can use masked links (e.g. `[text](https://site.com)`) in the message content.
+</Alert>
+
+## Fetching and deleting responses
+
+In addition to replying to a slash command, you may also want to delete the initial reply. You can use <DocsLink type="class" parent="ChatInputCommandInteraction" symbol="deleteReply" brackets /> for this:
+
+<CH.Code lineNumbers={false}>
+
+```js focus=2
+await interaction.reply('Pong!');
+await interaction.deleteReply();
+```
+
+</CH.Code>
+
+Lastly, you may require the _`Message`_ object of a reply for various reasons, such as adding reactions. You can use the <DocsLink type="class" parent="ChatInputCommandInteraction" symbol="fetchReply" brackets /> method to fetch the _`Message`_ instance of an initial response:
+
+<CH.Code lineNumbers={false}>
+
+```js focus=2:3
+await interaction.reply('Pong!');
+const message = await interaction.fetchReply();
+console.log(message);
+```
+
+</CH.Code>
+
+Or, pass the _`fetchReply: true`_ option when sending the reply:
+
+<CH.Code lineNumbers={false}>
+
+```js
+const message = await interaction.reply(}content: 'Pong!', fetchReply: true });
```

Comment:
```suggestion
const message = await interaction.reply({ content: 'Pong!', fetchReply: true });
```

Reply:
Interesting how Prettier doesn't fail for this.